### DISCUSSION TITLE
Can I write a mobile app in Python?

### DISCUSSION THREAD

**Discussion by PrOkSSy** [community/community#44002] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

I am a beginner programmer and I would like to ask you how to make an application in Python and is it even possible?

**Comment by kalinin-egor** [comment, marked as answer]
Hey, hi!

you have framework: kivy.

With him you can right mobile apps on python

good luck!

**Reply by PrOkSSy** [reply]
Thanks!
